src/Persistence/MsappArchive.cs

```diff
@@ -317,4 +286,15 @@ public void Dispose()
     }
 
     #endregion
+
+    public static class Directories
+    {
+        public const string SrcDirectory = "Src";
```

Comment:
If class name is Directories. There is no need to add "Directory" to each field

Reply:
right, I was going to do that too!